template/ProjectTemplate/samples/ProjectTemplateCustomSample.xaml

```diff
@@ -3,7 +3,7 @@
       xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
       xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-      xmlns:labs="using:CommunityToolkit.Labs.WinUI"
+      xmlns:labs="using:CommunityToolkit.WinUI.Controls"
```

Comment:
Should we change `xmlns:labs` here as well?

Do we go back to old `xmlns:controls` that we kind of aligned on for old toolkit e.g. past `xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"`?

Or do we do something new like `xmlns:ctwuc` which would be accurate abbreviation, but ugly?

Thinking maybe just `xmlns:controls`? Makes it simpler for folks that upgrade from 7.x->8.0 to align conventions?

Reply:
Intellisense would use `xmlns:controls` as the default I think? That + the above points, I'm sold on using that. I'll get on that.